src/utils/metadata/translate.rs

```diff
@@ -12,7 +13,7 @@ pub struct TranslateMetadata {
     /// before current statement in order to be correct.
     pub stmt_queue: VecDeque<String>,
     /// The name of the function that is currently being translated.
-    pub fun_name: Option<(String, usize, usize)>,
+    pub fun_name: Option<FunctionName>,
```

Comment:
The function's name is not the only thing being stored here, so it should be renamed. 

My proposal is the "FunctionMeta" type for the "fun_meta" field with the comment of "The metadata of the function…".

Reply:
Renamed.